_opensearch/snapshots/searchable_snapshot.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: default
+title: Searchable snapshots
+parent: Snapshots
+nav_order: 40
+has_children: false
+---
+
+# Searchable snapshots
+
+Searchable snapshots is an experimental feature with OpenSearch 2.4. Therefore, we do not recommend the use of this feature in a production environment. For updates on progress or if you want leave feedback that could help improve the feature, see the [searchable snapshot GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/2919). 
+{: .warning }
```

Comment:
Is this github issue #2919 the right issue to leave feedback? I prefer to submit new issue, may be with predefined template.

Reply:
the issue number came from @andrross. if there is a new issue number for feedback, let me know and I'll update it.

Reply:
Maybe something like the following will be better:

For updates on progress follow us on [GitHub](https://github.com/opensearch-project/OpenSearch/issues/3739) and if you have any feedback please [submit a new issue](https://github.com/opensearch-project/OpenSearch/issues/new/choose).